src/main/resources/templates/rooms/checkIn.html

```diff
@@ -96,13 +96,13 @@ <h3 class="content-header form-header">Mailadresse eingeben</h3>
                     <label for="email-auto-sign-in">Für künftigen automatischen Check-in speichern</label>
                 </div>
             </form>
-            <div  th:if="${privileged}">
+            <!--<div  th:if="${privileged}">
```

Comment:
Den Code kannst du komplett löschen, falls wir ihn nochmal brauchen kann mans ja in der git history finden.